coopaname_custom/models/hr_employee.py

```diff
@@ -43,6 +43,26 @@ def create(self, values):
             values["work_phone"] = _format_phone_number(values["work_phone"])
 
         employee = super().create(values)
+        if not employee.address_home_id:
+            partner = self.env["res.partner"].search(
+                [
+                    ("email", "=", employee.work_email),
+                    ("is_company", "=", False),
+                ],
+                limit=1,
+            )
+            if len(partner) == 1:
+                employee.address_home_id = partner
```

Comment:
could just be `if partner:`

Reply:
That what I did first, but it didn't work - see my comments on Frama/codeureuses yesterday: if `search()` doesn't find anything, it returns an empty recordset, which has boolean value True (!) (and doesn't return False).

Reply:
There must be something else going on, an empty recordset evaluates to False. I've used that everywhere and checked it with:

```python
bool(self.env["res.partner"].search([("name", "=", "crazy-name")]))
> False
```

Reply:
Fixed!